protos/commands.proto

```diff
@@ -17,49 +17,102 @@ enum Type {
 	ARTIST_COUNT = 8;
 	FOLDER_COUNT = 9;
 	SELECT_QUERY = 10;
-	CONTENT_QUERY = 11;
 	WHEN_DEVICE = 12;
```

Comment:
Add a `reserved` tag now that we skipped `11`:

https://developers.google.com/protocol-buffers/docs/proto#reserved